src/main/java/org/altbeacon/beacon/service/scanner/DistinctPacketDetector.java

```diff
@@ -1,37 +1,40 @@
 package org.altbeacon.beacon.service.scanner;
 
-import android.util.Log;
+import android.support.annotation.NonNull;
 
 import java.nio.ByteBuffer;
 import java.util.HashSet;
 import java.util.Set;
 
 /**
  * Created by dyoung on 4/8/17.
- *
+ * </p>
```

Comment:
You are correct. I believe the older javadoc tools automatically added `<p>` tags for each paragraph but not the associated closing tag. Which is why I just added the closing tags. And then checking some other code auto formatted through older Android Studio it used `<p/>`. Now it looks like the latest AS now simply uses `<p>`.

I'll update. Thanks for catching this.